test/XRP/fakes/fake-xrp-protobufs.ts

```diff
@@ -194,27 +288,58 @@ paymentAllFields.setPathsList(pathList)
 
 /**
  * Helper function to generate Transaction objects with the standard values from Payment objects.
+ * There must be at most one of accountSet or payment.
  *
+ * @param accountSet -AccountSet object to insert into the transaction.
  * @param payment -Payment object to insert into the transaction.
  * @returns Payment Transaction with the included payment param.
  */
-function buildStandardTransactionFromPayment(payment: Payment): Transaction {
+function buildStandardTestTransaction(
```

Comment:
as this grows, another option is: let this take two arguments: first one a `TransactionType` enum as you will find in the xpring-sdk.  the second one the actual transaction data, which can be typed as `TransactionData` with a type alias like `TransactionData = Payment | AccountSet | TrustSet | etc.` (whatever the exact typescript is for this).  Then switch case on the `TransactionType`.